server/boot/story.js

```diff
@@ -104,6 +105,23 @@ module.exports = function(app) {
     );
   }
 
+  function userStories({ body: { search = '' } = {} }, res, next) {
+    return app.dataSources.db.connector
```

Comment:
You still need to end early to prevent empty searches 
```js
  if (!search || typeof search !== 'string') {
       return res.sendStatus(404);
   }
```